.github/workflows/test.yaml

```diff
@@ -52,3 +52,41 @@ jobs:
       run: pipx run twine check --strict dist/*
     - name: Pip check
       run: pip check -v
+
+  visual-regression-tests:
+    name: Visual Regression Tests ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Print basic Python info
+      shell: python
+      run: |
+        import sys
+        print(sys.executable, sys.version, sep='\n')
+    - name: Install docrepr
+      run: python -m pip install --upgrade .[test,visual_tests]
+    - name: List packages in environment
+      run: pip list
+    - name: Install Firefox
+      run: python -m playwright install firefox
+    - name: Run tests
+      shell: bash
+      run: python -I -bb -X dev -W error -m pytest --compare-screenshots --asyncio-mode=auto
```

Comment:
We can move the `asyncio-mode` option to the `pytest.ini` file once we remove Python 3.6 tests (this option makes Python 3.6 tests fail)